tensorflow_addons/examples/Normalizations.ipynb

```diff
@@ -0,0 +1,126 @@
+{
```

Comment:
<p>Please move this to `addons/examples/jupyter_notebooks` <a href="https://github.com/tensorflow/probability/tree/master/tensorflow_probability/examples/jupyter_notebooks" target="_blank">as is done in tensorflow probability</a></p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/144/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>